modules/apps/headless/headless-asset-library/headless-asset-library-test/src/testIntegration/java/com/liferay/headless/asset/library/resource/v1_0/test/BaseAssetLibraryResourceTestCase.java

```diff
@@ -284,7 +284,8 @@ public void testPatchAssetLibraryBySite() throws Exception {
 			randomPatchAssetLibrary, expectedPatchAssetLibrary);
 
 		AssetLibrary getAssetLibrary =
-			assetLibraryResource.getAssetLibraryBySite(null);
+			assetLibraryResource.getAssetLibraryBySite(
```

Comment:
This will conflict with my PR since I've removed this test class by now

Reply:
well, let's see when it's on master :D

This is autogenerated soWe can not undo this on this PR